features/step_definitions/user_registration_steps.rb

```diff
@@ -0,0 +1,17 @@
+Given(/^the following users exists$/) do |table|
+  table.hashes.each do |hash|
+      User.create!(hash)
```

Comment:
1. We don't use methods with a `!` unless there´s a really good reason. What would be the reason here? 
2. There's a gem called FactoryGirl installed. We used that gem in the Rails Messaging Legacy Code Challenge. Why not use it here?